ml-agents/mlagents/trainers/torch/networks.py

```diff
@@ -546,7 +546,7 @@ def forward(
 
 
 class SimpleActor(nn.Module, Actor):
-    MODEL_EXPORT_VERSION = 3
+    MODEL_EXPORT_VERSION = 4
```

Comment:
I don't think we need to duplicate the comments here, but we should leave a comment that refers to the enum in BarracudaModelParamLoader.cs for a description of the versions.